test/JDBC/upgrade/13_6/verification_cleanup/14_3/BABEL-3121-vu-verify.sql

```diff
@@ -0,0 +1,44 @@
+USE db_babel_3121;
```

Comment:
why do we have 14_3 verification script inside 13_6 upgrade directory?

Reply:
removed